std/hash/poseidon_bn254.asm

```diff
@@ -43,11 +43,15 @@ machine PoseidonBN254(LASTBLOCK, operation_id) {
 
     // Whether the current round is a partial round
     pol constant PARTIAL = [0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0]*;
+
+    // Utility method needed until the parser can parse large numbers outside the current field.
```

Comment:
Still think it's not obvious why the numbers are outside the field, but I don't know a concise way of putting it and I think this will be reverted soon anyway, right?